Fronted/pet-feeder/src/pages/LogIn/LogIn.js

```diff
@@ -65,9 +67,7 @@ const LogIn = () => {
         <section>
           <h1 >You are logged in!</h1>
```

Comment:
Consider omitting the "You are logged in!" message when using useNavigate for instant redirection, as it may be visible only for seconds before the redirect or not even visible, since the user is now being automatically redirect on login successful.

If you want to display it do it as a notification or something